lib/wsdl.js

```diff
@@ -1130,7 +1130,7 @@ WSDL.prototype._processNextInclude = function(includes, callback) {
     includePath = url.resolve(self.uri, include.location);
   }
 
-  open_wsdl(includePath, this.options, function(err, wsdl) {
```

Comment:
> open_wsdl is used in lib/soap, tests started to fail as I modified the original function, thoughts on current changes?

I generally do not prefer exposing internal functions for test purposes. It seems to me we should fix `open_wsdl` instead of creating a new function.  Does `open_wsdl` even make sense as a name if it can't handle recursive wsdls?  Seems we should rename it to `open_non_recursive_wsdls`.